src/main/java/com/github/gokolo/personaldashboard/servlet/ListUsersServlet.java

```diff
@@ -14,26 +13,21 @@
 import com.github.gokolo.personaldashboard.dao.UserDAOImpl;
 import com.github.gokolo.personaldashboard.dto.UserDTO;
 
+/**
+ * @author Grace_Okolo
+ *
+ */
 @SuppressWarnings("PMD")
-@WebServlet(name = "ListUsersServlet", urlPatterns = "/xxx")
+@WebServlet(name = "ListUsersServlet", urlPatterns = "/users")
 public class ListUsersServlet extends HttpServlet {
     private UserDAO userDAO = new UserDAOImpl();
 
     public void doGet(final HttpServletRequest request, final HttpServletResponse response)
             throws IOException, ServletException {
         response.setContentType("text/html");
```

Comment:
Leftover code.
Since you are using the JSP technology to create your HTML output, there is no need to manually set the Content-Type any more.